app/routes/meetings/components/MeetingAnswer.tsx

```diff
@@ -1,52 +1,62 @@
+import { useHistory } from 'react-router-dom';
+import Button from 'app/components/Button';
 import LoadingIndicator from 'app/components/LoadingIndicator';
-import type { User } from 'app/models';
+import type { MeetingsTokenResponse } from 'app/reducers/meetingsToken';
+import type { ID } from 'app/store/models';
+import { MeetingInvitationStatus } from 'app/store/models/MeetingInvitation';
+import type { PublicUser } from 'app/store/models/User';
 
 type Props = {
-  response: string | null | undefined;
-  user: User;
-  status: number;
+  response: MeetingsTokenResponse;
+  user: PublicUser;
+  status: MeetingInvitationStatus;
   resetMeetingsToken: () => void;
-  meeting: number;
-  router: /*TODO: Router*/ Record<string, any>;
+  meeting: ID;
+};
+
+const statusTexts: { [value in MeetingInvitationStatus]: string } = {
+  [MeetingInvitationStatus.Attending]: 'skal nå delta',
+  [MeetingInvitationStatus.NotAttending]: 'skal nå ikke delta',
+  [MeetingInvitationStatus.NoAnswer]: 'har nå ikke svart på om de skal delta',
 };
 
 const MeetingAnswer = ({
   response,
   user,
   meeting,
   status,
-  router,
   resetMeetingsToken,
 }: Props) => {
+  const history = useHistory();
+
   if (!response) {
     return <LoadingIndicator loading />;
   }
 
   const handleLink = () => {
-    router.push(`/meetings/${meeting}`);
+    history.push(`/meetings/${meeting}`);
     resetMeetingsToken();
   };
 
   if (response === 'SUCCESS') {
-    const statusText = ['', 'Delta', 'Ikke delta'][status];
+    const statusText = statusTexts[status];
     return (
       <div
         style={{
           textAlign: 'center',
         }}
       >
         <h1>
-          {' '}
           Du har nå svart på invitasjonen{' '}
           <span role="img" aria-label="happy">
             😃
           </span>
         </h1>
         <p>
-          {user.firstName} skal nå {statusText} på møtet!
+          {user.firstName} {statusText} på møtet!
         </p>
         <p>
-          Logg inn og sjekk møtet <button onClick={handleLink}> her</button>
+          <Button onClick={handleLink}>Logg inn og sjekk møtet</Button>
```

Comment:
Kind of picky but I'd like this `Button` to be of type `dark` so that it matches the other "Log in" buttons.